src/manager.cpp

```diff
@@ -214,13 +233,115 @@ void Manager::deleteSingleFruVpd(
     logging::logMessage(std::string(i_dbusObjPath));
 }
 
+bool Manager::isValidUnexpandedLocationCode(
+    const std::string& i_unexpandedLocationCode)
+{
+    if ((i_unexpandedLocationCode.length() <
+         constants::UNEXP_LOCATION_CODE_MIN_LENGTH) ||
+        ((i_unexpandedLocationCode.compare(0, 4, "Ufcs") !=
+          constants::STR_CMP_SUCCESS) &&
+         (i_unexpandedLocationCode.compare(0, 4, "Umts") !=
+          constants::STR_CMP_SUCCESS)) ||
+        ((i_unexpandedLocationCode.length() >
+          constants::UNEXP_LOCATION_CODE_MIN_LENGTH) &&
+         (i_unexpandedLocationCode.find("-") != 4)))
+    {
+        return false;
+    }
+
+    return true;
+}
+
 std::string Manager::getExpandedLocationCode(
-    const sdbusplus::message::object_path& i_dbusObjPath)
+    const std::string& i_unexpandedLocationCode,
+    [[maybe_unused]] const uint16_t i_nodeNumber)
```

Comment:
const uint16_t i_nodeNumber = 0 
?

Reply:
Default parameter can be declared either in function definition or declaration. Not in both places